src/Stryker.Core/Stryker.Core/TestRunners/VsTest/RunEventHandler.cs

```diff
@@ -46,6 +46,10 @@ public RunEventHandler(AutoResetEvent waitHandle, ILogger logger, string runnerI
 
             TimeOut = testRunCompleteArgs.IsAborted;
 
+            TimeOut = testRunCompleteArgs.IsAborted;
```

Comment:
```suggestion
```

Reply:
Yep
For some reason,  my repo is out of sync and leads to difficult merge
sorry for that,